src/main/java/calculator/App.java

```diff
@@ -0,0 +1,12 @@
+package calculator;
+
+public class App {
+    public static void main(String[] args) {
+        String input = InputView.inputExpression();
+
+        StringCalculator calculator = new StringCalculator();
+        int result = calculator.calculate(input);
```

Comment:
```suggestion
        int result = StringCalculator.calculate(input);
```
StringCalculator가 상태 값이 없어 클래스 메서드로 구현해도 되지 않을까?